packages/chart/src/components/Legend/helpers/createFormatLabels.tsx

```diff
@@ -93,31 +109,21 @@ export const createFormatLabels =
       return reverseLabels(seriesLabels)
     }
 
-    // DEV-4161: replaceable series name in the legend

```

Comment:
Why was this removed?

Reply:
replaced it with shorter verstion  if (config.series.some(item => item.name)) {
      const uniqueLabels = Array.from(new Set(config.series.map(d => d.name || d.dataKey))).map((val, i) => ({
        datum: val,
        index: i,
        text: val,
        value: colorScale(val)
      }))
      return reverseLabels(uniqueLabels)
    }